.github/workflows/cd.yml

```diff
@@ -1,5 +1,8 @@
 name: cd
 
+env:
```

Comment:
neat: Should we have the `env` key after the `on` and/or `permissions` keys as in other workflows? I think that `on` is more important for workflow understanding so it should go first.

Reply:
Yep, my bad, gonna fix it :)